packages/jest-resolve/src/defaultResolver.ts

```diff
@@ -5,62 +5,19 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import {dirname, isAbsolute, resolve as pathResolve} from 'path';
 import {fileURLToPath} from 'url';
-import pnpResolver from 'jest-pnp-resolver';
 import {
-  type SyncOpts as UpstreamResolveOptions,
-  sync as resolveSync,
-} from 'resolve';
-import * as resolve from 'resolve.exports';
-import {
-  findClosestPackageJson,
-  isDirectory,
-  isFile,
-  readPackageCached,
-  realpathSync,
-} from './fileWalkers';
```

Comment:
do we still need `fileWalkers` if `unrs-resolver` does file caching?

Reply:
There are still other references, maybe for 3rd-party resolvers?

By the way, we have another `defaultAsyncResolver` by using `unrs-reslver`'s async resolution? Maybe in another PR.